packages/dds/tree/src/shared-tree-core/sharedTreeCore.ts

```diff
@@ -361,8 +363,9 @@ export class SharedTreeCore<TEditor extends ChangeFamilyEditor, TChange>
 			idCompressor: this.idCompressor,
```

Comment:
Would it be possible to reduce the duplication between the implementations of `processCore` and `processMessagesCore` by having the former call the latter?

Reply:
Yes, good catch. Changed it.